File: JavaFundamentals/src/BasicSyntaxConditionalStatementsAndLoops/Exercise/Division.java
package BasicSyntaxConditionalStatementsAndLoops.Exercise;

import java.util.Scanner;

public class Division {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int number = Integer.parseInt(scanner.nextLine());
        int divisible = 0;

        if (number % 10 == 0) {
            divisible = 10;
        } else if (number % 7 == 0) {
            divisible = 7;
        } else if (number % 6 == 0) {
            divisible = 6;
        } else if (number % 3 == 0) {
            divisible = 3;
        } else if (number % 2 == 0) {
            divisible = 2;
        }

        if (divisible == 0) {
            System.out.println("Not divisible");
        } else {
            System.out.printf("The number is divisible by %d", divisible);
        }
    }
}

File: JS Fundamentals/Basic Syntax, Conditional Statements and Loops - Exercise/division.js
function divisible(numb) {
    numb = Number(numb)
    if (numb % 10 === 0) {
        console.log('The number is divisible by 10')
    } else if (numb % 7 === 0) {
        console.log('The number is divisible by 7')
    } else if (numb % 6 === 0) {
        console.log('The number is divisible by 6')
    } else if (numb % 3 === 0) {
        console.log('The number is divisible by 3')
    } else if (numb % 2 === 0) {
        console.log('The number is divisible by 2')
    } else {
        console.log('Not divisible')
    }
}

divisible(30);
divisible(15);
divisible(12);
divisible(1643);